WMF Framework/ReadingListsSyncOperation.swift

```diff
@@ -661,7 +661,6 @@ internal class ReadingListsSyncOperation: ReadingListsOperation {
         }
         let group = WMFTaskGroup()
         var remoteEntriesToCreateLocallyByArticleKey: [String: APIReadingListEntry] = [:]
-        var requestedArticleKeys: Set<String> = []
```

Comment:
During debugging I noticed that `requestedArticleKeys` isn't used anywhere so I took the liberty of removing it to unclutter the code. Please take a look to be double sure 🙏

Reply:
`requestedArticleKeys` ensures the same article summary request isn't made twice. For example, if this function is importing the same article to two different lists (which presents as two separate entries), the app only needs to get the summary for that article once.

Reply:
Oh yes, I read the code wrong :+1: Thanks

Reply:
No problem, I wrote it and still had to do a double take to ensure it was still used. Open to suggestions for improving readability.